mlir/test/Dialect/Vector/vector-transfer-unroll.mlir

```diff
@@ -2,363 +2,373 @@
 // RUN: mlir-opt %s -test-vector-transfer-unrolling-patterns=reverse-unroll-order --split-input-file | FileCheck %s --check-prefix=ORDER
 
 // CHECK-LABEL: func @transfer_read_unroll
```

Comment:
CHECK-LABEL and CHECK-SAME seem to be repeated in this file quite a lot. Consider using a "shared" label like here:
* https://github.com/llvm/llvm-project/blob/main/mlir/test/Dialect/Vector/linearize.mlir